### PR TITLE
Update Gradle Wrapper to 8.9

### PR DESCRIPTION
**Pull request by wzieba**
This small PR updates Gradle Wrapper to 8.8.

#### Testing

Green CI checks should be enough to verify that this change works as expected.

### CODE REVIEW

**Comment by wzieba** [comment]
Sorry, I mixed up commits here. The PR is now ready for review @ParaskP7

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.2.1-all.zip
+distributionSha256Sum=f8b4f4772d302c8ff580bc40d0f56e715de69b163546944f787c87abf209c961
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.8-all.zip
```
Comment (ParaskP7) [inline gradle/wrapper/gradle-wrapper.properties]: Suggestion (💡): A newer version of Gradle is now available, the [8.9](https://docs.gradle.org/8.9/release-notes.html), which brings error/warning reporting, IDE integration and other improvements, along with some bug fixes, maybe we should now consider updating to that instead? 🤔

**Review by ParaskP7 — APPROVED** [review]
👋 @wzieba !

I have reviewed this PR, everything works as expected, good job! 🌟 

-----

I have left a suggestion (💡) comment for you to consider. I am going to approve this PR anyway, since none is blocking. I am **NOT** going to merge this PR yet to give you some time to apply any of my suggestions. However, feel free to ignore them and merge the PR yourself.

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (wzieba) [reply gradle/wrapper/gradle-wrapper.properties]: Good idea, thanks! Addressed in d4ecb111465b2312b3a503f1977a9f08101dc250

Comment (ParaskP7) [reply gradle/wrapper/gradle-wrapper.properties]: Awesome, thanks @wzieba , I did another check and everything LGTM, double-approved!

**Pull request merged**